cpplint.py

```diff
@@ -1111,21 +1111,18 @@ def ProcessCategory(category):
           error(filename, linenum, 'readability/nolint', 5,
                 f'Unknown NOLINT error category: {category}')
 
-def ProcessGlobalSuppresions(lines):
-  """Deprecated; use ProcessGlobalSuppressions."""
-  ProcessGlobalSuppressions(lines)
-
-def ProcessGlobalSuppressions(lines):
+def ProcessGlobalSuppressions(filename: str, lines: list[str]) -> None:
```

Comment:
If we want to keep supporting Python 3.8 then the first line of code in this file must be:
`from __future__ import annotations`